bodhi/tests/server/test_util.py

```diff
@@ -33,6 +34,16 @@
 from bodhi.tests.server import base
 
 
+# get bleach version
+bleach_v = bleach.__version__.split('.')
+for idx, val in enumerate(bleach_v):
+    try:
+        val = int(val)
+    except ValueError:
+        pass
+    bleach_v[idx] = val
```

Comment:
Maybe you can use `pkg_resources.parse_version()`

Reply:
done, and updated this commit with the change.

https://github.com/fedora-infra/bodhi/pull/3500/commits/751e80ff94681e7c997e509771e284a5a2d178a1